canarytokens/tokens.py

```diff
@@ -598,6 +599,24 @@ def _get_response_for_fast_redirect(
             return redirectTo(redirect_url.encode(), request)
         return GIF
 
+    @staticmethod
+    def _get_info_for_idp_app(request: Request):
+        src_data = {}
+        saml_request = prepare_request(request)
+        if saml_request:
+            identity = extract_identity(saml_request)
+            src_data["identity"] = identity
+        http_general_info = Canarytoken._grab_http_general_info(request=request)
+        return http_general_info, {"src_data": src_data}
+
+    @staticmethod
+    def _get_response_for_idp_app(canarydrop: canarydrop.Canarydrop, request: Request):
+        if canarydrop.redirect_url:
+            if canarydrop.browser_scanner_enabled:
+                return Canarytoken._get_response_for_slow_redirect(canarydrop, request)
+            return Canarytoken._get_response_for_fast_redirect(canarydrop, request)
+        return Canarytoken._get_response_for_web(canarydrop, request)
+
```

Comment:
```suggestion
        if not canarydrop.redirect_url:
            return Canarytoken._get_response_for_web(canarydrop, request)
        if not canarydrop.browser_scanner_enabled:
            return Canarytoken._get_response_for_fast_redirect(canarydrop, request)
        
        return Canarytoken._get_response_for_slow_redirect(canarydrop, request)
```

I think it makes it a bit easier to understand if we reduce the indentation.